suite-native/module-send/src/sendFormThunks.ts

```diff
@@ -275,7 +290,7 @@ export const calculateCustomFeeLevelThunk = createThunk(
         }
 
         const feeLevels = response.payload;
-        dispatch(sendFormActions.storeDraft({ accountKey, tokenContract, formState: draftCopy }));
+        // dispatch(sendFormActions.storeDraft({ accountKey, tokenContract, formState: draftCopy }));
```

Comment:
Should this be commented? I it is not being used, please, delete

Reply:
Forgotten leftover, removed in 991fea8f54